### PR TITLE
Minor updates & Removal of Saveable 

### PR DESCRIPTION
**Pull request by sasana-uwu**
removed saveable and unified to using savable, corrected minor mistakes in constructor

### CODE REVIEW

**Comment by imnnos** [comment]
Fisher should probably be removed as a reviewer if you want this merged in time.

**Comment by sasana-uwu** [comment]
> Prof. Fisher should probably be removed as a reviewer if you want this merged in time.

done :)

File: src/Objects/IBoard.java [hunk 1]
```diff
@@ -1,12 +1,21 @@
+<<<<<<< HEAD:src/Objects/IBoard.java
```
Comment (tomking) [inline src/Objects/IBoard.java:1]: Looks like you guys pulled some difftool comments into this here

File: src/Objects/Piece.java [hunk 2]
```diff
@@ -27,7 +27,17 @@ public Piece(String id, int x, int y, ArrayList<Rule> rules, Hashtable<String, S
         this.image = image;
     }
 
+<<<<<<< HEAD:Objects/src/main/java/objects/Piece.java
```
Comment (tomking) [inline src/Objects/Piece.java:30]: Difftool comments here too

**Review by tomking — CHANGES_REQUESTED** [review]
You need to remove the difftool lines that you incorporated into your code.

**Review by tomking — APPROVED** [review]
Alright go for it

**Pull request merged**